lld/test/COFF/weak-ref-cycle.test

```diff
@@ -0,0 +1,42 @@
+REQUIRES: x86
```

Comment:
I think COFF doesn't have ELF's weak references. COFF only has weak externals, similar to ELF's weak definitions.

I'd use `weak-external-cycle.test` to be beside `weak-external{,2,3}.test`

Reply:
Renamed in the new version.